testing/lightspeed/test.py

```diff
@@ -83,66 +82,90 @@ def load_test_prepare():
 
     return None
 
+protos_path = WISDOM_PROTOS_SECRET_PATH
+s3_creds_model_secret_path = WISDOM_SECRET_PATH / "s3-secret.yaml"
+quay_secret_path = WISDOM_SECRET_PATH / "quay-secret.yaml"
+dataset_path = WISDOM_SECRET_PATH / "llm-load-test-dataset.json"
+s3_creds_results_secret_path = WISDOM_SECRET_PATH / "credentials"
+
+test_namespace="wisdom"
+tester_imagestream_name="llm-load-test"
+tester_image_tag="wisdom-ci"
+
+def deploy_and_warmup_model(replicas):
+    run.run(f"./run_toolbox.py wisdom deploy_model {replicas} \
+        {s3_creds_model_secret_path} \
+        {quay_secret_path} \
+        {protos_path} \
+        {tester_imagestream_name} \
+        {tester_image_tag} \
+        --namespace='{test_namespace}' \
+        --")
+
+        # Warmup
+    run.run(f"./run_toolbox.py wisdom warmup_model {protos_path} \
+        {tester_imagestream_name} \
+        {tester_image_tag} \
+        --namespace='{test_namespace}'")
```

Comment:
you need to use `./run_toolbox.py from_config` here, that will turn these calls into:
```suggestion
    run.run(f"./run_toolbox.py from_config wisdom deploy_model")

    # Warmup
    run.run(f"./run_toolbox.py from_config wisdom warmup_model")
```
and the mapping between the config file and the command parameters will be defined in `command_args.yaml`.

if you have some parameters that cannot be be passed via `command_args.yaml` (eg, because there's a loop defining multiple values to use), then use this syntax:
```
extra = {}

extra["instance_type"] = cfg["node"]["instance_type"]
extra["scale"] = cfg["node"]["count"]

run.run(f"./run_toolbox.py from_config cluster set_scale --extra \"{extra}\"")
```